website/src/blogs/2018-08-21-building-persistent-storage-with-openebs-and-local-ssds-on-gke.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Building persistent storage with OpenEBS and Local SSDs on GKE
+author: Ranjith Raveendran
+author_info: Ranjith is working as a Software Engineer at MayaData and working in the OpenEBS project. In his free time, he listens to music, watches movies, and goes for bike riding.
+date: 21-08-2018
+tags: OpenEBS, Solutions, Stateful Applications
+excerpt: In this blog post, we’ll discuss the advantages of using GKE Local SSDs instead of GPDs and how to set up persistent storage using OpenEBS for highly available storage volumes.
+---
+
+If you are currently using Google Cloud for your Kubernetes needs, it is more than likely that you are using Google’s Kubernetes service, GKE. The good news is that you have another option for persistent storage apart from Google Persistent Disks, or GPDs.
+
+> *With OpenEBS software, you can easily set up persistent storage using the Local SSDs of GKE instances.*
+
+In this blog post, we’ll discuss the advantages of using GKE Local SSDs instead of GPDs and how to set up persistent storage using OpenEBS for highly available storage volumes. Though GPDs are extremely easy to provision and natively integrated with GKE, there are some inconveniences with GPDs related to stateful applications.
+
+## Running Stateful Apps using GPD Volumes
+![GPDs as persistent disks may not be readily available](https://lh5.googleusercontent.com/JOkShLc8RiH5a-bWvFPqNWn9rs24C3toodcI3i_g7vew2-4FzYdL_-vX7X9nofXCwTMU0P9zVraY6NSRCIE5jfdlrIeXXSW7KuT74dPv7QnGn6oVRdbAKN1U9GkR-UdzDyOTPMqvoxX1EoNzIQ)
+
+When a GKE node goes down, a new node comes up as part of Cluster Autoscaler. GPD disks that are associated with the old node must be detached from the old node and attached to the new node. This must be done through a manual procedure. Even if automated detaching were available, it would still slow the system.
+
+- Performance of the GPD volumes is limited as they are part of a larger shared storage network. GKE is not able to make use of faster disks such as SSDs.
+- Slow fail-over essentially means no High Availability.
+- Poor I/O, unless unused disk space is not an issue.
+
+## Restrictions for Using Local SSDs as -is for Kubernetes
+
+Google cloud documentation on the restrictions for using local SSDs for Kubernetes is mentioned [here](https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/local-ssd).
+![Use of local disks has restrictions on GKE](https://lh4.googleusercontent.com/uE8p6fJtr2AYnqYAuv_4XWEf5ZQ1lgZ8p-wthognqxV2ayxGQf2iCh7C6LEK5qM_OwvrnDx3R4D9BaiQLac8LE1fmqm3u1BI_bDFYn2c3T-zc1cFmuvL-0WVZoAbHsv4Vo0elHttSxmdQv9OUw)
```

Comment:
try to add a blank line before and after the image, it will help keep the structure